justfile

```diff
@@ -1,3 +1,5 @@
+set dotenv-load := true
```

Comment:
Is this telling just to load a `.env` file?

Reply:
Yeah, it doesn't change anything, but just is deprecating loading .env by default, so recent just warns loudly about it, which was annoying